### ISSUE TITLE
Release master with react/promise 2.*

### ISSUE THREAD

**Issue by netiul** [reactphp/stomp#41]
Since #33 react/promise v2 is supported. Can this be officially released?

**Comment by OndraM** [comment]
Yes please, could we get a new tagged version? Cc @cboden @igorw 

Thanks :-)

**Comment by OndraM** [comment]
With recent merge of the close event, would you please consider tagging a new release? Would be perfect for us!

cc @WyriHaximus

**Comment by WyriHaximus** [comment]
@ondram looking into that now, but first figuring out the promise version bump
